src/ClusterBootstrap/template/RepairManager/latency-config.yaml

```diff
@@ -0,0 +1 @@
+hours_until_alert_expiration: {{cnf['repair-manager']['latency_rule']['hours_until_alert_expiration']}}
```

Comment:
This is a pretty long name. `alert_expiry`? Hashtag comment it's in hours should be sufficient.